### DISCUSSION TITLE
Tweak the look and behavior of ALT-TAB (next/previous window)

### DISCUSSION THREAD

**Discussion by roundwheel** [lxqt/lxqt#2514] [Q&A]
Is it possible to change the appearance of the window switcher that appears in the left of the screen when you hit ALT-TAB or ALT-SHIFT-TAB? I find it very difficult to determine which window I am looking for when only 5 of potentially many more windows initially appear on the left. I also find it a poor user experience when the whole screen switches through every window until you get to the one you want. Small icons in the center of the screen, and not switching windows until you select the desired icon makes for a much more user friendly experience, IMO. Can this look and behavior be tweaked? Or can I install a different window switcher (not sure what these are called)?

**Comment by tsujan** [comment]
The switcher is provided by the window manger; isn't it? So, it depends on the WM you use.

**Reply by yvon-comeau-reeldata** [reply]
I'm currently using KWin. I'll explore other options. Thank you!

**Reply by tsujan** [reply]
Yes, KWin has several ways of alt-tabbing. I don't remember how I'd set it up, but it shows windows beside each other and highlights the active one. I also limited it to the windows of the current workspace.

EDIT:
Actually, I use two "hot corners" instead of alt-tabbing: one for the current workspace, the other for showing all workspaces.

**Reply by stefonarch** [reply]
There is a simple switcher for kwin providing just "Small icons in the center of the screen, and not switching windows until you select the desired icon". Unfortunately I don't remember how it was called.

**Reply by tsujan** [reply]
KDE's config is a haystack in which you need to find a needle. The good news is that the needle will be found if you're patient. It might take 5 minutes to an hour though.

**Reply by yvon-comeau-reeldata** [reply]
Found it!

![image](https://github.com/lxqt/lxqt/assets/128511627/9fd7122c-3e2b-4369-871e-f812b28622b9)
